numpy/core/src/multiarray/methods.c

```diff
@@ -1003,19 +1003,26 @@ any_array_ufunc_overrides(PyObject *args, PyObject *kwds)
     int i;
     int nin, nout;
     PyObject *out_kwd_obj;
+    PyObject *fast;
     PyObject **in_objs, **out_objs;
 
     /* check inputs */
     nin = PyTuple_Size(args);
     if (nin < 0) {
         return -1;
     }
-    in_objs = PySequence_Fast_ITEMS(args);
+    fast = PySequence_Fast(args, "could not convert args for fast access");
```

Comment:
Be nice to make all the error messages the same. Maybe `"Cannot convert args for fast access"` or some such if length is a problem.

Reply:
Missed one in `ufunc_override.c`.

Reply:
sorry, fixed that too